front/app/components/CustomFieldsForm/Fields/RankingField/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+
+import { Controller, useFormContext } from 'react-hook-form';
+
+import { IFlatCustomField } from 'api/custom_fields/types';
+
+import Ranking from './Ranking';
+import Error, { TFieldName } from 'components/UI/Error';
+import { CLError, RHFErrors } from 'typings';
+
+interface Props {
+  question: IFlatCustomField;
+}
+
+const RankingField = ({ question }: Props) => {
+  const {
+    control,
+    formState: { errors: formContextErrors },
+  } = useFormContext();
+
+  const name = question.key;
+
+  const errors = formContextErrors[name] as RHFErrors;
```

Comment:
Did we test this is working? With more complex question types sometimes we need to change the error handling but nit sure if it's the case here.

Reply:
Yeah it works, but I just noticed I messed up the formatting of the message. Just fixed that :+1: